server/routes.go

```diff
@@ -1343,6 +1348,88 @@ func (s *Server) PsHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, api.ProcessResponse{Models: models})
 }
 
+func (s *Server) InfoHandler(c *gin.Context) {
+	runnerMap := runners.GetAvailableServers(s.runnerDir)
+	runners := []string{}
+	for k := range runnerMap {
+		runners = append(runners, k)
+	}
+	sort.Strings(runners)
+
+	sysInfo := discover.GetSystemInfo()
+	ms, _ := Manifests()
+	fsUsed := uint64(0)
+	for _, m := range ms {
+		fsUsed += uint64(m.Size())
+	}
```

Comment:
I'm pretty sure this is double counting shared layers, so this will need more work.